test/utils.sh

```diff
@@ -1199,3 +1199,40 @@ retry() {
       sleep "${interval}"
   done
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# If the image is an index (multi-arch), replace the digest with the 0th manifest's digest; if it's a single-arch image manifest, leave it unchanged.
+resolve_arch_specific_digest() {
+  local FBC_FRAGMENT_IMAGE="$1"
```

Comment:
Following on the above comment, there is nothing special about this function that ties it to a FBC fragment image. That is just how we anticipate this function to be primarily used.